src/exberry_int/integration_exberry.py

```diff
@@ -62,15 +63,18 @@ def integration_exberry_main(
     env: 'ExberryIntegrationEnv',
     events: 'IntegrationEvents'):
 
-    outbound_queue = asyncio.Queue()
+    outbound_queue = asyncio.PriorityQueue()
     session_started = asyncio.Event()
 
-    LOG.info(f"Preparing session...")
-    outbound_queue.put_nowait(make_order_book_depth())
-
-    async def enqueue_outbound(msg: dict):
+    async def enqueue_outbound(msg: dict, priority: int = 1):
```

Comment:
You have `0` and `1` as priority levels in the code. Please use a symbolic constant that indicates the significance of these levels.

Reply:
Good idea, fixed